### PR TITLE
Added claifying documentation around GES configuration

### PR DESCRIPTION
**Pull request by davidjray**
Small tweak to documentation

### CODE REVIEW

**Comment by davidjray** [comment]
It's worth noting that validation already exists when configuring new `github_enterprise` services to prevent duplicate `repo_domain`

File: docs/get-started.md [hunk 1]
```diff
@@ -252,6 +254,10 @@ Here's an example command to configure a `github_enterprise` service:
 $ cci service connect github_enterprise enterprise --token <access token> --username <username> --email <email> --repo_domain <repo url>
 ```
 
+#### Limitations of _GitHub Enterprise Server Support_
+
+The GitHub Enterprise Server Support feature only allows one `github_enterprise` service to be configured per repository domain. This means the access token you are using must have permission to access all required resources at a given repository location.
```
Comment (jstvz) [inline docs/get-started.md:259]:
```suggestion
The GitHub Enterprise Server service only allows one `github_enterprise` service to be configured per server domain. This means the access token you are using must have permission to access all required resources at a given repository location.
```

**Review by jstvz — APPROVED** [review]
Some copyedits and tweaks.

---

Now that I'm reading this, I'm having second thoughts about calling it `repo_domain`, as it's really more of a `server_domain` or a `ges_domain`. 😬

**Review by prescod — CHANGES_REQUESTED** [review]
I'm moving this out of Approved status given James' suggestions.

**Comment by davidjray** [comment]
I actually like the idea of renaming `repo_domain` to `server_domain`, it is much clearer.

**Review by jstvz — APPROVED** [review]

**Pull request merged**